doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -397,16 +397,18 @@ This resource allows you to obtain a list of subresources published by the API:
 
 ## Query Entities [/v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
 
-### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
+### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,georel,geometry,attrs,options}]
 
 + Parameters
     + q: temperature==22.5;humidity==60 (optional, string) - General query parameter that allows
     to filter by attribute content
-    + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates. Comma separated values.
-    The number of values will depend on the 'geometry' attribute.
-    + geometry: circle;radius=10000 (optional, string) - Geospatial area for the query. 
-    The 'exterior' modifier allows to indicate that the referred area is the outside
-    of the region, for instance, 'polygon;exterior'
+    + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates of a reference geometry used for the query.
+    Comma separated values. The number of values and its interpreation depends on the `geometry` attribute.
```

Comment:
nit: interpreation -> interpretation

In addition, note that we are using `;` to separate each coordinate pairs (e.g. https://github.com/telefonicaid/fiware-orion/blob/develop/test/functionalTest/cases/1038_geo_point_support/geoquery_polygon_geopoint.test#L115). Maybe that should be also clarified here?